src/Fantomas/TriviaHelpers.fs

```diff
@@ -39,6 +39,14 @@ module internal TriviaHelpers =
             | TriviaNodeType.Token (_, tok) when (RangeHelpers.``range contains`` range t.Range) -> Some(tok, t)
             | _ -> None)
 
+    let ``keyword token after start column and on same line`` (range: range) (trivia: TriviaNode list) =
+        trivia
+        |> List.choose (fun t ->
+            match t.Type with
+            | TriviaNodeType.Token (_, tok) when (range.StartLine = t.Range.StartLine
```

Comment:
Does it need to handle cases where `in` is on another line?
```fsharp
do
    let _ = ()
      in
     () // note the different indent is allowed here due to `in` use
```

Reply:
Hmm, I think it should since we are doing this.
Thanks for pointing that out.